lab_6_pipeline/pos_frequency_pipeline.py

```diff
@@ -2,7 +2,14 @@
 Implementation of POSFrequencyPipeline for score ten only.
 """
 from typing import Optional
+from pathlib import Path
+from core_utils.article.article import Article
+from lab_6_pipeline.pipeline import CorpusManager, ConlluSentence, ConlluToken
 
+class EmptyFileError(Exception):
```

Comment:
are you planning for 10? if not, changes here are not required

Reply:
I hope i will do the  lab for 10, but i'm not sure....